docs/en/sql-reference/sql-functions/table-functions/generate_series.md

```diff
@@ -27,6 +27,7 @@ generate_series(start, end [,step])
 - `step`: the value to increment or decrement, optional.  Supported data types are INT, BIGINT, and LARGEINT. If not specified, the default step is 1. `step` can be either negative or positive, but cannot be zero.
 
 The three parameters must have the same data type, for example, `generate_series(INT start, INT end [, INT step])`.
+It supports named arguments from v3.3, where all parameters are input with names, like `name=>expr`, for example, `generate_series(start=>3, end=>7, step=>2)`.
 
```

Comment:
Do we have any requirement for the expr in named arguments?

Reply:
no, the same with positional arguments before.